authors/apps/authentication/validators.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+from django.shortcuts import get_object_or_404
+
+from rest_framework import serializers
+
+from authors.apps.authentication.models import User
+
+def validate_email(email):
```

Comment:
@wasibani-roy , There are duplicates in `validate_email` and `validate_username` refactor them by creating another function. This will maintain our maintainability

Reply:
The method validate_username was no longer in use I have removed that function because we worked with cryce and took out the duplications